giskard-demo-notebook/notebook/Email Classification Model.ipynb

```diff
@@ -909,51 +909,49 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "cTuCGaP2ZZnI",
     "pycharm": {
      "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
     "def predict(test_dataset):\n",
+    "    original_ds_len = len(test_dataset)\n",
+    "    assert type(test_dataset) == pd.core.frame.DataFrame\n",
+    "    test_dataset= test_dataset.squeeze() if len(test_dataset) >1 else test_dataset\n",
+    "    X_test = list(test_dataset)\n",
```

Comment:
out of curiosity, it doesn't work with pandas dataframe?

Reply:
It was giving. wrong results for the pandas dataframe :(